### ISSUE TITLE
Add class="notranslate" to meta tag

### ISSUE THREAD

**Issue by bcoelhonqda** [artesaos/seotools#139]
Hi,

I don't want the Google translate the meta tags, so i need add class="notranslate" attribute to the meta tags. It's possible?

Like this
`<title class="notranslate">Page title</title>`

**Comment by peterangelov** [comment]
Fast (maybe little bit dirty) solution? If you don't want to modify this package (as not all who use it might want this feature) you could modify the output in the template.

```php
{!! str_replace('<title>', '<title class="notranslate">', SEOMeta::generate()) !!}
```

**Comment by peterangelov** [comment]
@vinicius73 hi, I have prepared fix for this but I don't know how to test it properly. If the addition of "notranslate" class is controlled by config value, I can not change it during unit test, as the config values are set when SEOMeta class is added to service container as singleton. Any idea?

**Comment by vinicius73** [comment]
Closed by #178
